web/text_layer_builder.js

```diff
@@ -79,22 +81,23 @@ var TextLayerBuilder = (function TextLayerBuilderClosure() {
      *   for specified amount of ms.
      */
     render: function TextLayerBuilder_render(timeout) {
-      if (!this.textContent || this.renderingDone) {
+      if (!this.textContentStream || this.renderingDone) {
```

Comment:
This is basically making this whole file unusable together with `setTextContent`.
I'm assuming that you what you really want is something like this:
```javascript
if (!(this.textContent || this.textContentStream) || this.renderingDone) {
```